src/core/components/array-model.jsx

```diff
@@ -21,7 +21,8 @@ export default class ArrayModel extends Component {
     let { getComponent, getConfigs, schema, depth, expandDepth, name, specPath } = this.props
     let description = schema.get("description")
     let items = schema.get("items")
-    let title = schema.get("title") || name
+    //let title = schema.get("title") || name
```

Comment:
Do we still want this commented out here? If so, can you please add a note as to why it was commented out, and why you'd want to uncomment it?

Reply:
I believe title is the preferred property to be used, it's a display value that can be set to whatever we'd like in the swagger file, but Goa currently generates a long string of text, something like `"Mediatype identifier: application/vnd.goa.mytype; view=default"` - we could re-enable title if at some point Goa generates a better display value there for the models